src/coredump/coredumpctl.c

```diff
@@ -50,6 +50,9 @@
 #include "user-util.h"
 #include "util.h"
 
+static usec_t arg_since, arg_until;
+static bool arg_since_set, arg_until_set;
```

Comment:
the booleans should be initialized explicitly to false.

That said, I think it would make sense to remove the booleans and simply initialize arg_since and arg_until to the special value USEC_INFINITY, and then check explicitly for that.